bin/configs/spring-boot-reactive-serverSentEvents.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: spring
+outputDir: samples/server/petstore/springboot-reactive-serverSentEvents
```

Comment:
can you please add the new output folder to the github workflow so that it will be tested moving forward?

https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-spring.yaml
https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-spring-jdk17.yaml

Reply:
Added.

Fixed the email as well, thanks for pointing